sycl/include/CL/sycl/detail/pi.h

```diff
@@ -446,17 +446,17 @@ pi_result piextGetDeviceFunctionPointer(
 //
 // Context
 //
-pi_context piContextCreate( // TODO: change interface to return error code instead
+pi_result piContextCreate(
   const cl_context_properties * properties, // TODO: untie from OpenCL
-  pi_uint32         num_devices,
-  const pi_device * devices,
-  void (*           pfn_notify)(
-    const char * errinfo,
-    const void * private_info,
-    size_t       cb,
-    void *       user_data),
-  void *            user_data,
-  pi_result *       result);
+  pi_uint32        		num_devices,
+  const pi_device * 		devices,
+  void (*pfn_notify)(
+    const char 		      * errinfo,
+    const void 		      * private_info,
```

Comment:
Please, do not mix tabs and spaces in the same file.
I assume clang-format can fix this for the whole patch.

Reply:
Clang-format will change the whole file and all declarations, which is why I refrained from using it.
The real changes will get missed in the review in that case.
I saw this function declaration not as clearly aligned, so I changed the alignment. 
I can change it back if you think that is needed. 
I would like to add a separate commit for just the clang-format usage on the pi files.

Reply:
There are scripts for clang format that allow you to:
1. Run clang format on the patch: http://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting
2. Use clang format in editors so it's possible to apply clang format just for one line only:
https://clang.llvm.org/docs/ClangFormat.html#vim-integration

Reply:
I use 2.) for my development. For this file, I chose to follow the format in the file.
I will check method 1) and use it in the future.
For this patch, I'll remove this alignment change and upload it again.
Hopefully testing infrastructure will not give any flaky test failures and cause further delays.